### PR TITLE
Copy sources from Versions.props to NuGet.config

### PR DESCRIPTION
**Pull request by jcagme**
**TL;DR**

We need that all restore sources are located in `NuGet.config` since internal feeds can only be restored from there. More details including next steps [here](https://github.com/dotnet/arcade/blob/master/Documentation/RestoreSourcesUpdateStatus.md)

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -9,6 +9,8 @@
     <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
     <add key="arcade-validation" value="https://dotnetfeed.blob.core.windows.net/arcade-validation/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="arcade-validation" value="https://dotnet.myget.org/F/symreader-converter/api/v3/index.json" />
+    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```
Comment (riarenas) [inline NuGet.config]: nuget.org is already there, and arcade-validation is a key already

Comment (riarenas) [reply NuGet.config]: It also looks like we're missing  https://dotnet.myget.org/F/symreader/api/v3/index.json

Comment (jcagme) [reply NuGet.config]: Oh yeah, I was in the middle of the update when you dropped by and I almost got a heart attack :)

Comment (riarenas) [reply NuGet.config]: Always Ricardo's fault 😭

**Pull request merged**